core/components/molecules/popover/__stories__/variants/boundaryElement.story.jsx

```diff
@@ -38,25 +39,36 @@ export const boundaryElement = () => {
   const ref = React.useRef < React.HTMLDivElement > null;
 
   return (
-    <div ref={ref.current} style={{ height: 150, border: '1px dashed', padding: 20, overflow: 'auto' }}>
+    <div ref={ref.current} className="overflow-auto p-7 OuterWrapper">
```

Comment:
We require a scroll behaviour in this story to show that popover hides whenever user scrolls. Please add this behaviour in story